Mathlib/LinearAlgebra/BilinearForm/Basic.lean

```diff
@@ -46,236 +47,183 @@ Bilinear form,
 
 open BigOperators
 
-universe u v w
+open LinearMap (BilinForm)
 
-/-- `BilinForm R M` is the type of `R`-bilinear functions `M → M → R`. -/
-structure BilinForm (R : Type*) (M : Type*) [CommSemiring R] [AddCommMonoid M] [Module R M] where
```

Comment:
Can you leave behind a `@[deprecated] alias` for `_root_.BilinForm`?

Reply:
I tried adding:
```
@[deprecated] alias _root_.BilinForm := LinearMap.BilinForm
```
but then `open LinearMap (BilinForm)` no longer seems to work - I have to replace every `BilinForm` with `LinearMap.BilinForm`.

Reply:
Perhaps a better idea then is `export LinearMap (BilinForm)` at the top of the file, which means that any old code that uses `BilinForm` resolves it to `LinearMap.BilinForm` automatically (with no deprecation).

Reply:
Okay, done.